pontoon/base/tests/models/test_entity.py

```diff
@@ -397,3 +397,18 @@ def test_entity_project_locale_tags(entity_a, locale_a, tag_a):
 
     entities = Entity.for_project_locale(project, locale_a, tag=tag_a.slug,)
     assert entity_a not in entities
+
+
+@pytest.mark.django_db
+def test_get_or_create():
+    """ Test
+    """
+    created = True
+    assert created == True
+
+
+@pytest.mark.django_db
+def test_bulk_update():
+    """ Test
+    """
+    assert True
```

Comment:
I have this two empty tests for get_or_create and bulk_update accordingly. What about test data? I need EntityQuerySet to call both of them. How I can create it?

Reply:
I would add a test case for the `_get_word_count` method, to test if we compute the word count as expected.

Then, I'd use your two tests to see if the `word_count` gets assigned during their execution. The tests should be moved to `pontoon/base/tests/managers/test_entity.py`, where you can find some inspiration on how to write such tests.